frontend/workflows/serverexperimentation/src/tests/experiment-details.test.tsx

```diff
@@ -27,29 +26,6 @@ describe("Start Experiment workflow", () => {
     const component = shallow(
       <ExperimentDetails
         upstreamClusterTypeSelectionEnabled
-        hostsPercentageBasedTargetingEnabled={false}
-        onStart={() => {}}
-      />
-    );
-    expect(component.find(FormFields).dive().debug()).toMatchSnapshot();
-  });
-
-  it("renders correctly with host percentage based faults enabled", () => {
```

Comment:
these tests depend on `hostsPercentageBasedTargetingEnabled` attribute which was removed in https://github.com/lyft/clutch/pull/1500 - removing them.